cmd/joy/build.go

```diff
@@ -21,13 +21,15 @@ func NewBuildCmd() *cobra.Command {
 }
 
 func NewBuildPromoteCmd() *cobra.Command {
+	var chartVersion string
+
 	cmd := &cobra.Command{
 		Use:   "promote",
 		Short: "Promote a project to given version",
 		Long: `Promote a project to given version in given environment.
 Typically called at the end of a CI pipeline to promote a new build to default target environment.
 
-Usage: joy build promote <env> <project> <version>`,
+Usage: joy build promote [--chart-version <version>] <env> <project> <version>`,
```

Comment:
We need this feature because we create a new chart for every new deployment?

If not (or regardless), I would tweak the usage to specify the chart version may not be the same as the version. 

```
[--chart-version <chart-version>]
```

Reply:
Correct, we're publishing a new chart with every release of the service with how I've setup the pipeline right now, similar to how `joy-generator` is also setup.

Reply:
```suggestion
Usage: joy build promote [--chart-version <chart-version>] <env> <project> <version>`,
```